transport/http/jsonrpc/encode_decode.go

```diff
@@ -40,9 +40,9 @@ type EncodeResponseFunc func(context.Context, interface{}) (response json.RawMes
 // JSON encodes the object directly.
 type EncodeRequestFunc func(context.Context, interface{}) (request json.RawMessage, err error)
 
-// DecodeResponseFunc extracts a user-domain response object from an HTTP
-// request object. It's designed to be used in JSON RPC clients, for
+// DecodeResponseFunc extracts a user-domain response object from an JSONRPC
+// response object. It's designed to be used in JSON RPC clients, for
 // client-side endpoints. One straightforward DecodeRequestFunc could be
```

Comment:
Oops, looks like a copy/paste flub in the comment! Probably there are others of these, too. Also, does the actual comment from this point on make sense?

Reply:
Thanks for spotting that. I've made some tweaks to the doc comments in that file.